src/python/grpcio/grpc/_server.py

```diff
@@ -278,6 +278,11 @@ def set_trailing_metadata(self, trailing_metadata):
             self._state.trailing_metadata = trailing_metadata
 
     def abort(self, code, details):
+        # treat OK like other invalid arguments: fail the RPC
+        if code == grpc.StatusCode.OK:
+            logging.warning(
```

Comment:
I think logging.error fits the situation better. We consider passing OK a programming defect and we should complain as loudly as possible.